tools/integration-tests/tester/tests/testutil.go

```diff
@@ -262,13 +261,6 @@ func SendIotaTransaction(t *testing.T, from *framework.Peer, to *framework.Peer,
 
 	// sign transaction
 	var txn *transaction.Transaction
-	emptyID := identity.ID{}
-	if txConfig.AccessManaPledgeID == emptyID {
-		txConfig.AccessManaPledgeID = local.GetInstance().ID()
```

Comment:
instead of `local`, this should be the ID of the from peer, same below

Reply:
It's already being set to the ID of the peer in the test.
That's why this is deleted.

Reply:
ah, okay then